ci/hardware_tests/device/core.py

```
@@ -1,7 +1,132 @@
+        from trezorlib.models import ALL_MODELS
+        from trezorlib.transport.webusb import WebUsbTransport, dev_to_str
+
+        if path is None:
+            self.log("[hardware/usb] No target device known, skipping USB reset")
+            return
+
+        # Not `WebUsbTransport.enumerate(usb_reset=True)`: that resets every
+        # Trezor the process can see, and it reads a string descriptor per
+        # device, which raises for the very device we are trying to recover.
+        # Match on bus/port instead, skipping devices that cannot answer.
+        usb_ids = {id for model in ALL_MODELS for id in model.usb_ids}
```

Ruff (0.16.1)

[error] Variable `id` is shadowing a Python builtin

(A001)